packages/plugin-moxie-autonomous-trading/src/utils/utility.ts

```diff
@@ -194,6 +210,12 @@ const errorMessages: Record<string, string> = {
     AERR044: "The sell condition value must be a non-negative number",
     AERR045: "Number of users for sell condition is provided bigger than the actual users in the rule. Please provide lower number.",
     AERR046: "Number of users for sell condition is provided bigger than actual users in the group. Please provide lower number.",
+    AERR050: "At least one user must be selected to copy trades from.",
+    AERR051: "Please check if valid Senpi user is tagged. Each Senpi user id starts with the letter 'M'",
```

Comment:
I think the message here can be a bit more user friendly, since user won't have context on what are senpi user id
```suggestion
    AERR051: "Please check if valid Senpi user is tagged e.g. @[betashop.eth|M4].",
```

Reply:
fixed in 29ae226